scripts/fix-daily-branch

```diff
@@ -38,15 +38,20 @@ drop_cherry_picks_from_daily_branch() {
     gitcmd fetch ${remote}
     echo "Backing out all debian/patches/*cpick* from ${daily_branch} branch"
     gitcmd checkout -B ${daily_branch} ${source_branch}
-    gitcmd log --oneline -- debian/patches/*cpick* | cut -d" " -f1 | xargs git revert
-    # switch back to current development branch
-    cat <<EOF
+    cpick_files=$( ls debian/patches/*cpick* 2> /dev/null || echo "" )
```

Comment:
It might be easier to use `find` here, which wouldn't require the error handling:

```
$ find debian/patches -name "*cpick*"
$ echo $?
0
```